_includes/v22.2/ui/insights.md

```diff
@@ -0,0 +1,224 @@
+CockroachDB will then keep a streaming histogram in memory for each distinct statement fingerprint that has seen an execution latency longer than `sql.insights.anomaly_detection.latency_threshold`, and will flag any execution with a latency in the 99th percentile (greater than p99) for its fingerprint.
+
+Additional controls filter out executions that are less actionable:
+
+- The execution's latency must also be longer than twice the median latency (`> 2*p50`) for that fingerprint. This ensures that the elevated latency is significant enough to warrant attention.
+- The execution's latency must also be longer than `sql.insights.anomaly_detection.latency_threshold`. Some executions are slower than usual, but are still fast enough for the workload.
+
+The [`sql.insights.anomaly_detection.memory_limit` cluster setting]({{ link_prefix }}cluster-settings.html#setting-sql-insights-anomaly-detection-memory-limit) cluster setting limits the amount of memory available for tracking these streaming latency histograms. When this threshold is surpassed, the least-recently touched histogram is evicted. The default of `1 MiB` is sufficient for tracking about 1,000 fingerprints.
+
+You can track the `sql.insights.anomaly_detection.memory` and `sql.insights.anomaly_detection.evictions` [metrics]({{ link_prefix }}ui-custom-chart-debug-page.html) to determine if the settings are appropriate for your workload. If you see a steady stream of evictions or churn, you can either raise the `sql.insights.anomaly_detection.memory_limit` cluster setting, to allow for more storage; or raise the `sql.insights.anomaly_detection.latency_threshold` cluster setting, to examine fewer statement fingerprints.
+
+### Schema insights settings
+
+You can configure the index recommendations in the [**Schema Insights** tab](#schema-insights-tab), [**Explain Plans** tab]({{ link_prefix }}ui-statements-page.html#insights), and [**Databases** page]({{ link_prefix }}ui-databases-page.html) with the following [cluster settings]({{ link_prefix }}cluster-settings.html):
+
+| Setting                                                                | Default value | Description                                                                                                             | Where used                           |
+|------------------------------------------------------------------------|---------------|-------------------------------------------------------------------------------------------------------------------------|--------------------------------------|
+|`sql.metrics.statement_details.index_recommendation_collection.enabled` | `true`        | Whether or not index recommendations are enabled for indexes that could be or are used during statement execution.      | Schema Insights and Explain Plans tab|
```

Comment:
Same suggestion for this table as the one above: for each row I'd make the cluster setting name a link to the canonical doc for that setting, remove the repeated default value column, and make 'where used' link to the docs for that section of the UI